includes/class-content-toc.php

```diff
@@ -38,26 +38,7 @@ protected function __construct() {
 		// TOC shortcode counter
 		$this->id_counter = 0;
```

Comment:
@johnbillion I've noticed that `id_counter` is initialised in `__construct()`, but `$placeholder` while declaring the property https://github.com/dashaluna/hm-content-toc/pull/49/files#diff-a20892fca22525983b4f0537d0ea0ebaR29

What's the best approach here? Wondering if doing it in one style/in one place is easier to read.